rust/gui-client/src-tauri/src/client/gui.rs

```diff
@@ -127,7 +100,7 @@ pub(crate) fn run(cli: client::Cli) -> Result<(), Error> {
 
     let tray = SystemTray::new().with_menu(system_tray_menu::signed_out());
 
-    tracing::debug!("Setting up Tauri app instance...");
+    tracing::info!("Setting up Tauri app instance...");
```

Comment:
I wanted it because a user had a hang somewhere around here and if it happens again I want to be able to bisect it without asking them to be on `DEBUG` before their Client hangs